src/server/scripts/Commands/cs_modify.cpp

```diff
@@ -454,6 +468,48 @@ class modify_commandscript : public CommandScript
         return false;
     }
 
+    static bool CheckModifySpeed(ChatHandler* handler, char const* args, Unit* target, float& speed, float minimumBound, float maximumBound, bool checkInFlight = true)
+    {
+        if (!*args)
+        {
+            return false;
+        }
+
+        speed = double(std::atof(args));
```

Comment:
```suggestion
        speed = Acore::StringTo<float>(args).value();
```

Reply:
Try this

Reply:
See example https://github.com/azerothcore/azerothcore-wotlk/blob/888549bcdc2622a5f75cd1765e3915e17c9e8ead/src/server/scripts/Commands/cs_account.cpp#L93

Reply:
maybe .value will fix it, but just StringTo will fail. Will commit and let CI test